packages/remirror__messages/src/index.ts

```diff
@@ -27,4 +27,3 @@ export * as ExtensionTextHighlightMessages from './extension-text-highlight-mess
 export * as ExtensionUnderlineMessages from './extension-underline-messages';
 export * as ExtensionWhitespaceMessages from './extension-whitespace-messages';
 export * as ReactComponentMessages from './react-components-messages';
-export type { MessageDescriptor } from '@lingui/core';
```

Comment:
We should add a changeset for `@remirror/messages` since its public API has changed. If the only thing we remove is a TypeScript type, I guess we could bump a just minor version for this package.

Reply:
Yep, haven't added a changeset at all yet. But to be honest I was assuming _everything_ would be major bumped to avoid confusion?

Reply:
No, I think we need to bump everything because I've changed the API on the root `Remirror` React component, it now expects an `i18nFormat` function instead of a `i18n` prop which is a Lingui instance.

Even if we kept the prop name the same the prop value itself is completely different